sentry_sdk/serializer.py

```diff
@@ -275,19 +275,23 @@ def _serialize_node_impl(
                     break
 
                 str_k = text_type(k)
-                v = _serialize_node(
-                    v,
-                    segment=str_k,
-                    should_repr_strings=should_repr_strings,
-                    is_databag=is_databag,
-                    remaining_depth=remaining_depth - 1
-                    if remaining_depth is not None
-                    else None,
-                    remaining_breadth=remaining_breadth,
-                )
-                if v is not None:
```

Comment:
Instead of doing this if-else, could you just add `v` here unconditionally? `_serialize_node` of `None` is `None`

Reply:
thanks!